### PR TITLE
404

### PR DESCRIPTION
**Pull request by blukai**
image

### CODE REVIEW

File: assets/fonts/OpenSans/font-face.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@font-face {
```
Comment (howardchung) [inline assets/fonts/OpenSans/font-face.css]: Could we reuse the other font and save having to bundle all of the font files?

File: src/components/FourOhFour/FourOhFour.jsx [hunk 2]
```diff
@@ -3,8 +3,12 @@ import styles from './FourOhFour.css';
 
 export default () => (
   <div className={styles.container}>
-    <h1>404</h1>
-    <h2>Page not found.</h2>
-    :(
+    <img src="/assets/images/sad.gif" role="presentation" />
```
Comment (howardchung) [inline src/components/FourOhFour/FourOhFour.jsx]: do we have permissions to use this image?

File: assets/fonts/OpenSans/font-face.css (hunk 1 above)
Comment (blukai) [inline assets/fonts/OpenSans/font-face.css]: what do u mean?

File: src/components/FourOhFour/FourOhFour.jsx (hunk 2 above)
Comment (blukai) [inline src/components/FourOhFour/FourOhFour.jsx]: It's dota 2 image like a items, cosmetics, heroes. I am sure that there are no problems with it.

File: assets/fonts/OpenSans/font-face.css (hunk 1 above)
Comment (howardchung) [inline assets/fonts/OpenSans/font-face.css]: Seems excessive to me to add a whole set of font pages to be used for one page.

File: src/components/FourOhFour/FourOhFour.jsx (hunk 2 above)
Comment (howardchung) [inline src/components/FourOhFour/FourOhFour.jsx]: sounds good :)

File: assets/fonts/OpenSans/font-face.css (hunk 1 above)
Comment (blukai) [inline assets/fonts/OpenSans/font-face.css]: It's a light font, PT Sans have no light version, so I found another one. Also this font will be used on home page.

File: src/components/FourOhFour/FourOhFour.jsx (hunk 2 above)
Comment (blukai) [inline src/components/FourOhFour/FourOhFour.jsx]: I took it [here](https://github.com/rossengeorgiev/dota2_notebooks/blob/master/List%20Emoticons.ipynb). I'll try to ask him

Comment (blukai) [inline src/components/FourOhFour/FourOhFour.jsx]: @rossengeorgiev, Can we use the emoticon that you animated? 🦄

Comment (rossengeorgiev) [inline src/components/FourOhFour/FourOhFour.jsx]: I assume the file is taken from https://github.com/rossengeorgiev/dota2_notebooks/blob/master/List%20Emoticons.ipynb in which case, absolutely. Use at will

Comment (blukai) [inline src/components/FourOhFour/FourOhFour.jsx]: @rossengeorgiev, Cool, thank you very much!

File: assets/fonts/OpenSans/font-face.css (hunk 1 above)
Comment (howardchung) [inline assets/fonts/OpenSans/font-face.css]: I'm not too familiar with CSS, so correct me if I'm wrong: Can't we use the font-weight property to make the other font lighter?

File: src/components/FourOhFour/FourOhFour.jsx [hunk 3]
```diff
@@ -3,8 +3,12 @@ import styles from './FourOhFour.css';
...
+    <div className={styles.message}>
+      The page you’re looking for can’t be found.
```
Comment (howardchung) [inline src/components/FourOhFour/FourOhFour.jsx]: consider putting the strings in `lang/en.json`.  We don't HAVE to do it now, but anything we do now we don't have to do later.

File: assets/fonts/OpenSans/font-face.css (hunk 1 above)
Comment (blukai) [inline assets/fonts/OpenSans/font-face.css]: Sure if that has the necessary weight, PT Sans has only 400 (nornal) and 700 (bold). Open Sans that I added has 300 (light)

File: src/components/FourOhFour/FourOhFour.jsx (hunk 3 above)
Comment (blukai) [inline src/components/FourOhFour/FourOhFour.jsx]: I guess we need to split the lang file. Examle:
```
// tooltips
"icon_github": "View on GitHub",
...
// errors
"error": "Error",
..
// dota: runes
...
```

File: src/components/FourOhFour/FourOhFour.jsx [hunk 4]
```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import styles from './FourOhFour.css';
+import strings from 'lang/en.json';
```
Comment (howardchung) [inline src/components/FourOhFour/FourOhFour.jsx]: I think you can just do 'lang/en'.  ESLint wants absolute imports first.

File: src/lang/en.json [hunk 5]
```diff
@@ -168,5 +168,7 @@
   "parsing_by": "Parsing by",
   "cheese_by": "Cheese icon by",
   "app_name": "OpenDota",
-  "error_message": "Whoops! Something went wrong."
+  "error_message": "Whoops! Something went wrong.",
+  "error": "Error",
+  "four_oh_four_message": "The page you’re looking for can’t be found."
```
Comment (rossengeorgiev) [inline src/lang/en.json]: I suggest removing the contractions. `The page you are looking for cannot be found.`

**Pull request merged**